README.md

```diff
@@ -2,7 +2,11 @@
 
 ## About
 
-Repository for the visualization of the [CC-Catalog](https://github.com/creativecommons/cccatalog) data.
+The landscape of licensed content is wide and varied. We have domains linking to other domains, different license types, and some metadata. This information is extracted from the Internet every month by Common Crawl and is available as the [CC-Catalog](https://github.com/creativecommons/cccatalog). It is fair to mention that we have 250 million works and growing! The goal of this project is to show the users how licensers are connected, their licensed content, and show how the licensing wave is expanding.
```

Comment:
I think it would be better to use this post for the intro text since it is more suited to a wide audience: https://creativecommons.org/2020/01/23/introducing-the-linked-commons/

Reply:
> better to use this post for the intro text

@kgodey to be clear, I should add the first paragraph in the above blog post and add a link to the same for reference?

Reply:
I haven't taken a detailed look at the blog post recently. Please use your best judgement.